sky/cli.py

```diff
@@ -1699,36 +1718,43 @@ def autostop(
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
     """Schedule or cancel an autostop or autodown for cluster(s).
 
-    CLUSTERS are the name (or glob pattern) of the clusters to stop.  If both
+    CLUSTERS are the names (or glob patterns) of the clusters to stop. If both
     CLUSTERS and ``--all`` are supplied, the latter takes precedence.
 
-    If --down is passed, autodown (tear down the cluster; non-restartable) is
-    used, rather than autostop (restartable).
+    When multiple autostop settings are specified for the same cluster, e.g.,
+    using ``sky autostop`` or ``sky launch -i``, the last setting takes
+    precedence.
 
-    ``--idle-minutes`` is the number of minutes of idleness (no pending/running
-    jobs) after which the cluster will be stopped automatically.
-    Scheduling autostop twice on the same cluster will overwrite the previous
-    autostop schedule.
+    Idleness means there are no in-progress (pending/running) jobs in a
+    cluster's job queue (``sky queue``).
 
-    ``--cancel`` will cancel the autostopping. If the cluster was not scheduled
-    autostop, this will do nothing to autostop.
+    Idleness time of a cluster is reset to zero, when any of these happens:
```

Comment:
We also have the definition of `idleness time` in the help str of `--idle-minutes-to-autostop`. Should we change those as well?